### ISSUE TITLE
Preview of multiple selection options should list option names

### ISSUE THREAD

**Issue by kambala-decapitator** [xbmc/Official-Kodi-Remote-iOS#981]
Array indexes reflect the order that I selected the options. I'd also sort the array.

![image](https://github.com/xbmc/Official-Kodi-Remote-iOS/assets/1557784/f5dc1be6-d34d-48d8-a5ff-e9750a08f743)
![image](https://github.com/xbmc/Official-Kodi-Remote-iOS/assets/1557784/54351c71-0d34-45c7-b8f1-578f68ced57b)

**Comment by wutschel** [comment]
Agreed, did a local change for this already. But I would like to go through some other use cases and menus before I submit the PR.

**Comment by wutschel** [comment]
Yet open:

API response provides labels which could be used (same as are shown in the UI list). I guess it was done this way to limit the characters to be displayed. Same happens for the skip steps. It shows "-600, -300, ...", but the API also provides "-10 min" and "-5 min". But then the whole list would exceed the available space.

Solution should use comma-separated labels in a multiline label.
